README.md

```diff
@@ -3,9 +3,9 @@ SSDynamicText
 
 [![Circle CI](https://circleci.com/gh/splinesoft/SSDynamicText.svg?style=svg)](https://circleci.com/gh/splinesoft/SSDynamicText) [![codecov.io](http://codecov.io/github/splinesoft/SSDynamicText/coverage.svg?branch=master)](http://codecov.io/github/splinesoft/SSDynamicText?branch=master)
  
-iOS 7's [`UIFontDescriptor`](https://developer.apple.com/library/ios/documentation/UIKit/Reference/UIFontDescriptor_Class/) is pretty neat. Also pretty neat is dynamic text that responds to the preferred text size that the user specified in Settings.app.
+iOS 7's `UIFontDescriptor` is pretty neat. Also pretty neat is dynamic text that responds to the preferred text size that the user specified in Settings.app.
```

Comment:
I removed because below is a `UILabel`, `UIButton`, etc. and those objects have not link, so I though that everyone should have or no one.
But no problem. I'll restore link.